.github/workflows/package.yml

```diff
@@ -31,6 +31,16 @@ on:
         description: 'run id for build_starter.yml, search for all output'
         required: false
         type: string
+      platforms:
```

Comment:
Is it for test only?

Reply:
At the moment this is to passed into integration test workflow, I'm also look into package script to see if it can reduce some work based on platform and products